pages/dmp_supporting_info/support_00_rdm_terms.md

```diff
@@ -21,30 +21,45 @@ For Norwegian translations, take a look a the [NO-RDA Termliste for forskningsda
 
 ## Key research data management terms explained (alphabetical order, non-exaustive)
 
-### Archiving, Data archival
+### Archiving
+#### [CODATA RDM Terminology (version 2023)](https://doi.org/10.5281/zenodo.10626170) [![License: CC BY 4.0](https://img.shields.io/badge/License-CC_BY_4.0-lightgrey.svg)](https://creativecommons.org/licenses/by/4.0/)
+> Engage in curation activity that ensures that records, objects, metadata and data are properly
+selected, stored, and can be accessed, and for which logical and physical integrity are maintained
+over time, including security and authenticity
+
+### Data archive
+#### [CODATA RDM Terminology (version 2023)](https://doi.org/10.5281/zenodo.10626170) [![License: CC BY 4.0](https://img.shields.io/badge/License-CC_BY_4.0-lightgrey.svg)](https://creativecommons.org/licenses/by/4.0/)
+> Archive (noun): Curated collection or repository containing physical or digital static records, objects, metadata and
+data deemed suitable for permanent retention, set up and managed to established standards and
+models, such as ISAD(G), CoreTrustSeal, and the OAIS reference model, that ensure long term
+integrity, security, authenticity and accessibility of the records, objects, metadata and data.
 
 ### Data preservation
 #### [CODATA RDM Terminology (version 2023)](https://doi.org/10.5281/zenodo.10626170) [![License: CC BY 4.0](https://img.shields.io/badge/License-CC_BY_4.0-lightgrey.svg)](https://creativecommons.org/licenses/by/4.0/)
-Preservation: An activity within archiving in which specific items of data are maintained over time so that they can 
+> Preservation: An activity within archiving in which specific items of data are maintained over time so that they can 
```

Comment:
```suggestion
> Preservation: An activity within archiving in which specific items of data are maintained over time so that they can still be accessed and understood through changes in technology. 
```